CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# v2.3.4 (2024-08-19)
+
+## Build Changes
+* sundog: strip "settings" prefix from populated settings ([#94])
```

Comment:
```suggestion
* Fix a bug in `sundog` where settings would be rendered incorrectly from the "settings" prefix ([#94])
```

Reply:
> Fix a bug in `sundog` where settings would be rendered incorrectly from the "settings" prefix

It wasn't a rendering bug. It was more that it couldn't detect that settings were populated. But a higher-level description might be better:
> Fix a bug in `sundog` that caused it to regenerate populated settings